plugins/org.jkiss.dbeaver.net.ssh.ui/src/org/jkiss/dbeaver/ui/net/ssh/SSHTunnelConfiguratorUI.java

```diff
@@ -230,6 +234,29 @@ private static void setNumberEditStyles(Text text) {
         text.setLayoutData(gdt);
     }
 
+    @Nullable
+    private static DBPAuthInfo promptCredentialsDialog(@NotNull SSHConstants.AuthType type,
+        @NotNull DBWHandlerConfiguration configuration) {
+        final DBPAuthInfo credentialsToConnectTo;
+        try {
+            credentialsToConnectTo = DBWorkbench.getPlatformUI()
+                .promptUserCredentials(SSHUIMessages.model_ssh_dialog_credentials,
+                    SSHUIMessages.model_ssh_dialog_credentials_username,
+                    configuration.getUserName(),
+                    type.equals(SSHConstants.AuthType.PUBLIC_KEY)
+                    ? SSHUIMessages.model_ssh_dialog_credentials_passphrase
+                    : SSHUIMessages.model_ssh_dialog_credentials_password,
+                    configuration.getPassword(),
+                    type.equals(SSHConstants.AuthType.PUBLIC_KEY),
+                    false
+                );
+        } catch (Exception e) {
+            log.error(e);
```

Comment:
Uh, maybe we want to display the error in the UI instead of just logging and silently ignoring it?